### PR TITLE
docs: add text version of certificates tutorial

### PR DESCRIPTION
**Pull request by edibotopic**
The ADSys documentation contains one tutorial currently.

It is only presented in video format, and generally there should be a text-based version of any tutorial that is included in our documentation.

This PR provides a text-based version of the video tutorial. It attempts to adhere closely to the video content while adding/subtracting as necessary for clarity.

Some minor rewording is also done to the main and tutorial index pages to improve clarity and consistency.

UDENG-3603

### CODE REVIEW

File: docs/tutorial/certificates-autoenrolment.md [hunk 1]
```diff
@@ -1,5 +1,185 @@
...
+ping <IPv4-address-of-resource>
+```
+
+It should be possible to `ssh` into a machine on the network:
+
+```text
+ssh -i ~/.ssh/adsys-integration.pem root@<IPv4-address-of-resource>
+```
+
+For example, an instance of Ubuntu Mantic Minotaur (development branch) will give an output that shows it is running on Azure based on the kernel version:
```
Comment (jibel) [inline docs/tutorial/certificates-autoenrolment.md:181]: I'd replace `Ubuntu Mantic Minotaur (development branch)` by `Ubuntu 24.04 LTS`
Mantic was a development version and is now EOL.

File: docs/tutorial/certificates-autoenrolment.md [hunk 2]
```diff
@@ -1,5 +1,185 @@
...
+
+```text
+Welcome to Ubuntu Mantic Minotaur (development branch) (GNU/Linux 6.5.0-1004-azure x86_64))
```
Comment (jibel) [inline docs/tutorial/certificates-autoenrolment.md:184]: Same here, replace by
```
Welcome to Ubuntu 24.04 LTS (GNU/Linux 6.5.0-1004-azure x86_64)
```

(the kernel version number is probably inaccurate but it is not important)

File: docs/.custom_wordlist.txt [hunk 3]
```diff
@@ -75,6 +75,7 @@ LTS
 MacOS
 macOS
 manpages
+Mantic
```
Comment (jibel) [inline docs/.custom_wordlist.txt:78]: it can be removed once the version of the release is updated to use the LTS

**Review by jibel — CHANGES_REQUESTED** [review]
Thanks for the transcription of the demo in a tutorial. Only request is to change the version of Ubuntu to the latest LTS

**Comment by edibotopic** [comment]
thanks @jibel -- I was trying to maintain consistency with the video but I think it's better to update, as you suggest. Will do.

File: docs/tutorial/certificates-autoenrolment.md (hunk 1 above)
Comment (edibotopic) [reply docs/tutorial/certificates-autoenrolment.md]: done in [1270c01](https://github.com/ubuntu/adsys/pull/1073/commits/1270c01359b4840b6b57e6d06e7cfe0c867ab834)

File: docs/tutorial/certificates-autoenrolment.md (hunk 2 above)
Comment (edibotopic) [reply docs/tutorial/certificates-autoenrolment.md]: done in [1270c01](https://github.com/ubuntu/adsys/pull/1073/commits/1270c01359b4840b6b57e6d06e7cfe0c867ab834)

File: docs/.custom_wordlist.txt (hunk 3 above)
Comment (edibotopic) [reply docs/.custom_wordlist.txt]: I amended this changed and removed it from the wordlist after updating references to the Ubuntu LTS.

**Review by jibel — APPROVED** [review]
All good!

**Pull request merged**